cic_dev/addon/setup-aic

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# author: sundar.gnanasekaran@intel.com
+
+
+echo "Enter root user password:"
+read a
+
+export password=$a
+export AIC_WORK_DIR=${PWD}
+
+#Install Feature set packages / modules
+./pre-requisites/setup-feature
+
+#Initiate Early-Launch
+./pre-requisites/launch-early
+
+#Remove un-used apks
+./pre-requisites/remove-app
+
+#Install Kernel
+./pre-requisites/setup-kernel
+
+#Install CIC
+cd $AIC_WORK_DIR
+echo "Installing CIC -- START"
+./aic install -e -d none
```

Comment:
./aic install  -u -e -d none
Pls add -u to  update docker image and remove unnecessary apks

Reply:
I have added this change as well... :)